src/Components/WebAssembly/testassets/MonoSanity/MonoSanity.csproj

```diff
@@ -1,16 +0,0 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
I didn't bring this app forward. It seems low value and requires additional work to get the loader script functioning. Let me know if you feel strongly about keeping this.

Reply:
I think it's great to remove this 👍 

This project (and its tests) was created before Blazor, so we had some way to make sense of how .NET code should behave on WebAssembly, and so that if the runtime behaviors around low-level basics like exception handling, boxing, etc. varied then we'd have a way to observe that objectively.

By now, .NET on WebAssembly is well enough established that we don't have to get independent proof that the low-level basics work correctly!